components/style/values/computed/length.rs

```diff
@@ -188,6 +188,13 @@ pub enum LengthOrPercentage {
     Calc(CalcLengthOrPercentage),
 }
 
+impl Default for LengthOrPercentage {
```

Comment:
Why? I don't think this is necessarily a good default. Ultimately defaults should be per-property, so I think if something doesn't have a clear default we shouldn't implement it.